vision/getting-started/image_editing.ipynb

```diff
@@ -1,710 +1,1247 @@
 {
```

Comment:
<p>For mask dilation, it's unclear what 0 means and what 1 means. Does 0 mean no dilation and thus does not attempt to reduce artifacts? And does 1 mean the maximum dilation and it will attempt to reduce the most amount of artifacts it can?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/506/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Reply:
<p>Clarified now. 0 means no dilation, 1 means totally dilated aka the mask will cover everything. Editing with mask dilation 1 is essentially just generating an entirely new image. </p>